src/middlewared/middlewared/plugins/ldap.py

```diff
@@ -263,7 +263,6 @@ class Config:
         Int('dns_timeout', default=5),
         Int('kerberos_realm', null=True),
         Str('kerberos_principal'),
-        Bool('has_samba_schema', default=False),
```

Comment:
Just a question.  I believe removing this equates to a public API change, and can see the change in docs for the `do_update`.  Is this all that needs to occur for a public API change?

Reply:
Yes. That's why one needs to be somewhat careful about changing a public accepts decorator.